Isle of Trust/src/components/EndOfLevel.tsx

```diff
@@ -1,35 +1,66 @@
 import React from "react";
 import ReactDom from "react-dom";
 import { Link } from "react-router-dom";
+import "../css/EndOfLevel.css"
+import Confetti from 'react-confetti'
 
 //Renders the pop up window that appears after the level has been completed
 //TODO: This is a basic set up of the pop up using the logic from the "History" pop up, it still needs content
 //TODO: Maybe add confetti in the background using https://www.npmjs.com/package/react-confetti
 interface EndOfLevelProps {
     level: number;
+    success: boolean;
 }
-export function EndOfLevel({ level }: EndOfLevelProps) {
+export function EndOfLevel({ level, success }: EndOfLevelProps) {
+    const width =500;
+    const height = 600;
     function updateLevel() {
         level++;
     }
     return ReactDom.createPortal(
         <div className="popup-container">
             <div className="overlay"></div>
             <div className="popup">
-                <div className="popup-content">
-                    <h1 className="text-center">Congratulations!</h1>
-                    <h2 className="text-center">You finished Level {level}</h2>
-                    <Link
-                        reloadDocument
-                        className="link"
-                        to={"/level" + (level + 1)}
-                    >
-                        <button onClick={updateLevel}> Next Level</button>
-                    </Link>
-                </div>
+                {success ? (
+                        <div className="popup-content">
```

Comment:
this pop up has scroll bars on each axis which is a little weird. If you can figure out how to remove that, that'd be great but if it's too hard don't worry about it